package.json

```diff
@@ -18,7 +18,6 @@
     "@types/lodash": "^4.14.197",
     "@types/node": "20.5.4",
     "@types/react": "^18.2.21",
-    "autoprefixer": "^10.4.15",
```

Comment:
Dropped autoprefixier, `postcss-preset-env` contains it by default
https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#autoprefixer